### ISSUE TITLE
Error in prepareStackTrace

### ISSUE THREAD

**Issue by piscisaureus** [denoland/deno#620]
A failed assertion in unit_test.ts gives me incorrect line numbers, plus the following scary error:

```
=====Error inside of prepareStackTrace====
TypeError: CallSite method toString expects CallSite as receiver
    at Object.toString (<anonymous>)
    at stack.map (gen/bundle/main.js:126369:77)
    at Array.map (<anonymous>)
    at prepareStackTrace$1 (gen/bundle/main.js:126369:28)
    at Function.prepareStackTraceWrapper (gen/bundle/main.js:126358:18)
    at Object.runTests [as cb] (C:/Users/BertBelder/d/deno/js/testing/testing.ts:60:39)
=====Original error=======================
Error
    at Object.assert (eval at _gatherDependencies (gen/bundle/main.js:126631:16), <anonymous>:32:19)
    at makeTempDir (C:/Users/BertBelder/d/deno/js/unit_tests.ts:72:28)
    at Object.runTests [as cb] (C:/Users/BertBelder/d/deno/js/testing/testing.ts:55:23)
out\debug\deno.exe js/unit_tests.ts permW0N0
out\debug\deno.exe js/unit_tests.ts permW1N0 --allow-write
```

**Comment by hayd** [comment]
I think this is fixed now.

**Comment by kevinkassimo** [comment]
@hayd seems true, duplicate of #808 and got fixed in #809

**Comment by ry** [comment]
Thx
